SignalServiceKit/src/Devices/OWSRecordTranscriptJob.m

```diff
@@ -102,7 +102,7 @@ - (void)runWithAttachmentHandler:(void (^)(TSAttachmentStream *attachmentStream)
 
     // If there is an attachment + text, render the text here, as Signal-iOS renders two messages.
     if (attachmentsProcessor.hasSupportedAttachments && transcript.body && ![transcript.body isEqualToString:@""]) {
-        // render text *after* the attachment
+        // We want the text to appear after the attachment.
         uint64_t textMessageTimestamp = transcript.timestamp + 1;
```

Comment:
We often used `timestamp + 1` to place a new interaction directly after another to which it is related.

* Sometimes we forgot to add 1, I've tried to address that in this PR since we sometimes look up interactions by timestamp.